### PR TITLE
Update naming of kiwi configuration file: config.xml -> *.kiwi

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: xml/obs_package_formats.xml [hunk 1]
```diff
@@ -132,6 +132,12 @@ $PACKAGE.dsc
     </itemizedlist>
    </para>
    <para>
+    <note>
+     OBS only accepts files with a .kiwi suffix as a kiwi configuration file. Other
```
Comment (mschnitzer) [inline xml/obs_package_formats.xml]: Use `<filename class="extension">.kiwi</filename>`

Comment (mschnitzer) [inline xml/obs_package_formats.xml]: kiwi -> KIWI

File: xml/obs_package_formats.xml [hunk 2]
```diff
@@ -132,6 +132,12 @@ $PACKAGE.dsc
...
+     naming schemes like the config.xml KIWI supports are getting ignored in OBS.
```
Comment (mschnitzer) [inline xml/obs_package_formats.xml]: config.xml -> `<filename>config.xml</filename>

**Comment by bgeuken** [comment]
Adjusted. Thanks!

**Pull request merged**